process/block/metablock.go

```diff
@@ -64,6 +65,16 @@ func NewMetaProcessor(arguments ArgMetaProcessor) (*metaProcessor, error) {
 		return nil, err
 	}
 
+	resolver, err := arguments.ResolversFinder.IntraShardResolver(factory.MiniBlocksTopic)
```

Comment:
Actually in this case for miniBlocks I need the functionality of miniBlocksResolver/ResolversFinder which could resolve the request only from pool/storage because I don't want to request the given hashes from network. This mechanism is used for cleaning miniblocks pool from latest receives, and this action is done when the headers with nonces behind final are evicted from pool. In this point I want to check again the miniblocks pool and to remove all of them related to these headers if they are found there. If I use here the functionality of RequestHandler these miniblocks would be requested from network and this is not the intention in this case. I think that the refactor of this Request mechanism should be done in one separate task.

Reply:
I have created a task for this https://elrondnetwork.atlassian.net/browse/EN-5226

Reply:
ok